src/shell.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import cmd
+import os
+
+from src.password   import Password
+from src.utils      import *
+
+
+class Shell(cmd.Cmd):
+    prompt  = ALINEA
+    session = None
+
+    def __init__(self, session) -> None:
+        super().__init__()
+        self.session    = session
+        self.prompt     = self.session.prompt
+
+    def get_arguments(self, line):
+        arguments = {}
+        all_parameters = line.split()
+        arguments.update({'parameter' : all_parameters[0]})
+        options = []
+        for p in all_parameters[1:]:
+            if p.startswith('-') or  p.startswith('--'):
+                options.append(p)
```

Comment:
The loop above can be simplified into 
```
options = [
    p for p in all_parameters[1:] if p.startswith('-')
]
```

Reply:
The `startswith('--')` is not necessary because if it starts with `--` it also starts with `-` and the first condition is already true.